zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/IdentitySetupProcessors.java

```diff
@@ -28,6 +29,8 @@ public static void addIdentitySetupProcessors(
             ValueType.IDENTITY_SETUP,
             IdentitySetupIntent.INITIALIZE,
             new IdentitySetupInitializeProcessor(writers, keyGenerator))
-        .withListener(new IdentitySetupInitializer(securityConfig, config.isEnableIdentitySetup()));
+        .withListener(
+            new IdentitySetupInitializer(
+                securityConfig, config.isEnableIdentitySetup(), new AuthorizationConfigurer()));
```

Comment:
🔧 As.I understand the pattern here is to create any dependencies up in the `EngineProcessors` class i.e. [here](https://github.com/camunda/camunda/blob/main/zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/EngineProcessors.java#L119) and then pass that into the constructors.

I don't think it matters hugely here as the result is ultimately the same but it perhaps compliments consistency :)